src/odemis/acq/stream/_base.py

```diff
@@ -269,18 +269,18 @@ def __str__(self):
 
     def _getVAs(self, comp, va_names):
         if not isinstance(va_names, set):
-            raise ValueError("vas should be a set but got %s" % (va_names,))
+            raise ValueError(u"vas should be a set but got %s" % (va_names,))
```

Comment:
Capital letters

Reply:
As this commit is about changing the messages to unicode, it's orthogonal to the content of the messages. It's nicer to see each commit doing a single thing. However, I've added another commit, which does fix the capital for each message in acq.stream .